### PR TITLE
Reset simulator scales

### PR DESCRIPTION
**Pull request by Mazyod**
#304 was just so bad, sorry about that. Took the time to look at the code more, and I tested this code in `irb`.

### CODE REVIEW

File: lib/snapshot/runner.rb [hunk 1]
```diff
@@ -18,6 +18,13 @@ def work
 
       clear_previous_screenshots if Snapshot.config[:clear_previous_screenshots]
 
+      Helper.log.info "Resetting all simulators screen scale to 1".green
+
+      all_device_types = `xcrun simctl list devicetypes`.scan(/.*\s\((.*)\)/)
```
Comment (KrauseFx) [inline lib/snapshot/runner.rb]: Use `FastlaneCore::Simulator.all` to get a list of all available simulators. 

```ruby
FastlaneCore::Simulator.all.each do |d|
end
```

Here are the available attributes: https://github.com/fastlane/fastlane_core/blob/master/lib/fastlane_core/simulator.rb#L55-L59

Comment (Mazyod) [inline lib/snapshot/runner.rb]: @KrauseFx Done. Should I update https://github.com/fastlane/snapshot/blob/master/lib/snapshot/reset_simulators.rb#L29 as well?

**Comment by KrauseFx** [comment]
Ahhh, so sorry, I missed your pull request. Closing this for now, as it's already in the `master` branch. Thanks for contributing @Mazyod

**Comment by Mazyod** [comment]
Thanks for releasing v1.2.0! As long as its fixed, it's all good

**Pull request closed without merging**